src/machine/machine_atmega328p.go

```diff
@@ -467,3 +472,93 @@ var SPI0 = SPI{
 	sdo:  PB3,
 	sdi:  PB4,
 	cs:   PB2}
+
+// Pin Change Interrupts
+var (
+	pinCallbacks [3][8]func(Pin)
+	interrupts   [3]*interrupt.Interrupt
+	previous     [3]uint8
```

Comment:
It looks like `previous` is  never used?

Reply:
Ah, yeah, holdover from my _previous_  iteration... :-D